### PR TITLE
Status gateways

### PR DESCRIPTION
**Pull request by sbeaver-netgate**
Two tables eliminated
Colors defined to eliminate magic
Bootstrap column width specified
All obsolescent classes and elements removed

### CODE REVIEW

File: usr/local/www/status_gateway_groups.php [hunk 1]
```diff
@@ -54,113 +54,117 @@
...
 ?>
 
-<body link="#0000CC" vlink="#0000CC" alink="#0000CC">
-<?php include("fbegin.inc"); ?>
-	<table width="100%" border="0" cellpadding="0" cellspacing="0">
+<div class="table-responsive" id="mainarea">
+	<table class="table table-hover table-striped table-condensed">
 		<tr>
-		  <td>
+			<th class="col-md-1"><?=gettext("Group Name"); ?></th>
```
Comment (SjonHortensius) [inline usr/local/www/status_gateway_groups.php:66]: What's the reason you added column size-classes here? I'm pretty sure they're not needed

File: usr/local/www/status_gateway_groups.php [hunk 2]
```diff
@@ -54,113 +54,117 @@
...
+								} else {
+									$online = gettext("Gathering data");
+									$bgcolor = "#ADD8E6";  // lightblue
+								}
+?>
+								<td bgcolor="<?=$bgcolor?>">&nbsp;<?=htmlspecialchars($member)?>, <?=$online?>
+<?php
+							} else {
+?>
+						<td>&nbsp;
```
Comment (SjonHortensius) [inline usr/local/www/status_gateway_groups.php:145]: This `&nbsp;` isn't needed

**Comment by SjonHortensius** [comment]
You added status_gateway_groups.php to this PR as well; I think that's accidental, can you revert that?

**Comment by sbeaver-netgate** [comment]
I will attempt to do that.

## Steve

Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com | www.netgate.com

**Pull request closed without merging**